bitrise.yml

```diff
@@ -426,6 +426,18 @@ workflows:
     - swiftlint-extended@1:
         inputs:
         - linting_path: "$BITRISE_SOURCE_DIR"
+    - deploy-to-bitrise-io@1.9: {}
```

Comment:
Adding this deploy step allows us to keep the swiflint github check and so the available results when swiftlint fails

Reply:
See this commit where I forced a failure: https://github.com/mozilla-mobile/firefox-ios/runs/13033663063
And this commit fixing that failure: https://github.com/mozilla-mobile/firefox-ios/runs/13034848959